libbeat/outputs/elasticsearch/api.go

```diff
@@ -42,15 +41,6 @@ type CountResults struct {
 	Shards json.RawMessage `json:"_shards"`
 }
 
-func (r QueryResult) String() string {
```

Comment:
yup, only was used in unit/intgeration tests to compare the serialized json with some test string.